Resources/Prototypes/Damage/modifier_sets.yml

```diff
@@ -83,6 +83,15 @@
     Cold: 1.5
     Poison: 0.8
 
+- type: damageModifierSet
+  id: Lizard # Skin tougher, bones weaker, strong stomachs, cold-blooded (kindof)
+  coefficients:
+    Blunt: 1.1
+    Slash: 0.9
+    Cold: 1.5
+    Heat: 0.9
+    Poison: 0.9
```

Comment:
sus that this is in this PR, but if its gonna be then this should at least be renamed to `Scale` or something

Reply:
Since they're removed now I figured it wouldn't be too bad (ignore the fact I did this before they were removed)

Reply:
ye